File: Common/WPF Integration/IntegrationWithWPF/IntegrationWithWPFVB/RadUserControls/UserControl1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports Telerik.WinControls

Namespace RadUserControls
	Public Partial Class UserControl1
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub radButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles radButton1.Click
			RadMessageBox.Show("I am a RadButton")
		End Sub
	End Class
End Namespace

File: Common/WPF Integration/IntegrationWithWPF/IntegrationWithWPFCS/RadUserControls/UserControl1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls;

namespace RadUserControls
{
    public partial class UserControl1 : UserControl
    {
        public UserControl1()
        {
            InitializeComponent();
        }

        private void radButton1_Click(object sender, EventArgs e)
        {
            RadMessageBox.Show("I am a RadButton");
        }
    }
}